kyaml/yaml/rnode.go

```diff
@@ -718,10 +718,10 @@ func (rn *RNode) MustString() string {
 
 // Content returns Node Content field.
 func (rn *RNode) Content() []*yaml.Node {
-	if rn == nil {
```

Comment:
Please remain to check `if rn == nil`

Reply:
@koba1t do you still want this check retained with the knowledge that:

```
// YNode returns the yaml.Node value.  If the yaml.Node value is a DocumentNode,
// YNode will return the DocumentNode Content entry instead of the DocumentNode.
func (rn *RNode) YNode() *yaml.Node {
	if rn == nil || rn.value == nil {
		return nil
	}
```

will achieve the same?

Reply:
I think so.

Additionally, Please add a test to check `if rn == nil`.